Description: arm asm instruction
Forwarded: not-needed
--- a/art/runtime/interpreter/mterp/arm/main.S
+++ b/art/runtime/interpreter/mterp/arm/main.S
@@ -602,7 +602,7 @@ MterpCommonTakenBranch:
 #endif
     cmp     rPROFILE, #JIT_CHECK_OSR
     beq     .L_osr_check
-    subsgt  rPROFILE, #1
+    subgt   rPROFILE, #1
     beq     .L_add_batch                @ counted down to zero - report
 .L_resume_backward_branch:
     ldr     lr, [rSELF, #THREAD_FLAGS_OFFSET]
