Forwarded: not-needed
--- a/system/core/libutils/String8.cpp
+++ b/system/core/libutils/String8.cpp
@@ -26,6 +26,7 @@
 #include <ctype.h>
 
 #include <string>
+#include <limits>
 
 #include "SharedBuffer.h"
 
--- a/system/libziparchive/zip_archive_stream_entry.cc
+++ b/system/libziparchive/zip_archive_stream_entry.cc
@@ -23,6 +23,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 
+#include <limits>
 #include <memory>
 #include <vector>
 
