Description: Replace ssize_t with size_t to fix ftbfs
Forwarded: not-needed
--- a/system/libziparchive/zip_archive.cc
+++ b/system/libziparchive/zip_archive.cc
@@ -1370,7 +1370,7 @@ bool Reader::IsZeroCopy() const {
 }  // namespace zip_archive
 
 static std::span<uint8_t> bufferToSpan(zip_archive::Writer::Buffer buf) {
-  return {buf.first, ssize_t(buf.second)};
+  return {buf.first, size_t(buf.second)};
 }
 
 template <bool OnIncfs>
